src/pages/Home.js

```diff
@@ -9,7 +9,8 @@
   =========================================================
   * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 */
-import { useState } from "react";
+import { useState, useEffect } from 'react';
+import { useApi } from '../api';
```

Comment:
I had to change the import reference as follows to make it work or it wouldn't compile:

import { useApi } from '../hooks/useApi';

Reply:
Sorry I forgot to update the import, now it's correct